client/my-sites/checkout/checkout/wechat-payment-box.jsx

```diff
@@ -196,6 +201,18 @@ export class WechatPaymentBox extends Component {
 								<SubscriptionText cart={ cart } />
 							</span>
 							<div className="checkout__secure-payment">
+								{ moneyBackGuarantee && (
+									<div className="checkout__secure-payment-content">
+										<Gridicon icon="refresh" />
+										{ translate( ' 30-day Money Back Guarantee' ) }
```

Comment:
Hi! I've found a possible matching string that has already been translated **18** times:
`translate( 'All plans include a 30-day money-back guarantee.' )` *ES Score: 8*
See 1 additional suggestion in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/update/checkout-guarantee)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).